ToDos/ListTableViewController.swift

```diff
@@ -87,10 +100,36 @@ class ToDoTableViewCell: UITableViewCell {
         textLabel?.font = UIFont.boldSystemFont(ofSize: 30)
 
         // ask Joseph: why are the lines flashing?? why do they appear once I add more items?
-        
     }
     
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 }
+
+//    class CheckBox: UIButton {
```

Comment:
When a cell is clicked, it both unchecks/checks an item, and brings up the ToDoComposer for item editing. Trying to figure out how to make a button in a UITableViewCell that only takes up 1/8th of the cell....but not sure how to do that. I tried to make a new class CheckBox at the bottom of ListTableViewController, but not sure how to "plug that into" the TableViewCell.

Reply:
what's happening is you're trigging the composer to push on the `didTap` function. If you want different parts of your cell to do different things in response to a user interaction you'll probably need to ignore that function and do things manually from within the cell subclass. If you want to be able to navigate to the composer still you'll likely need to implement a delegate that can communicate actions from the cell back to the TableViewController. there are other ways to do it, but this is usually the clearest way.